man/deepsea-stage-5.7

```diff
@@ -0,0 +1,42 @@
+.TH deepsea 7
+.SH NAME
+DeepSea stage 5 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 5 (Removal)
+is a Salt orchestration that deploys the non-core services of ceph.
+.RE
+.PD
+.SH Preface
+.PP
+This stage is used to remove services from nodes.
+
+.B Services:
+.RS
+If you remove a service from your policy.cfg and apply it to the pillar by running stage.2 
+.B (see deepsea-stage-2 (7)) 
+running stage.5 will remove this service and all it's traces from the node.
```

Comment:
"it's" should be its, but could be reworded.... "remove all traces of the service from the node"